src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -64,6 +64,13 @@ class OptionsResolver implements Options
      */
     private array $nested = [];
 
+    /**
+     * BC layer. Remove in Symfony 8.0.
+     *
+     * @var array<string, true>
+     */
+    private array $deprecatedNested = [];
```

Comment:
same concern: nested is not a noun so this is confusing

Reply:
same as https://github.com/symfony/symfony/pull/59618#discussion_r1941384169

Reply:
renamed to `$deprecatedNestedOptions`